csharp/examples/IoTDataPipelineExample/Program.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Apache.Arrow.Ipc;
+using Apache.Arrow;
+
+namespace IoTPipelineExample
+{
+    class Program
+    {
+        public static int concurrencyLevel = 8;
+        public static int totalInputs = 10_000;
+        public static int queueCapacity = 1_000_000;
```

Comment:
These values are a bit confusing. If we are only going to create 10,000 inputs, why would we need a queue capacity of 1 million?

Reply:
@eerhardt,

Thanks for all your reviews, I have addressed them per your comments.